library/globals.inc.php

```diff
@@ -3509,6 +3509,20 @@ function gblTimeZones()
             '',
             xl('USPS Web Tools API Username')
         ),
+
+        'mdht_conformance_server_enable' => array(
+            xl('Use MDHT External Validation Service'),
+            'bool',                           // data type
+            '0',
+            xl('Enable CCDA conformance and validation API service')
+        ),
+
+        'mdht_conformance_server' => array(
+            xl('CCDA MDHT Validation API Server Address'),
+            'text',                           // data type
+            'http://ccda.healthit.gov',
```

Comment:
can we assume the file will always have the /referenceccdaservice/ path?  Or would it be better to specify the full server address here?  That leaves the option for people to customize / change this if they never need it to.

Reply:
all arguments are specific to this server.